Extensions/BBText/pixi-multistyle-text/README.md

```diff
@@ -0,0 +1,7 @@
+This extension is using a forked version of https://github.com/tleunen/pixi-multistyle-text, which adds the ability to handle bbcode styled tags.
+The PR adding this feature is in https://github.com/tleunen/pixi-multistyle-text/pull/78 and may become a part of the master branch.
```

Comment:
Let's hope we can get this merge to use an "official" version soon :)

Reply:
He seems quite positive about it at the PR :)